app/models/mixins/aggregation_mixin.rb

```diff
@@ -30,9 +35,11 @@ def aggregate_cpu_speed(targets = nil)
     aggregate_hardware(:hosts, :aggregate_cpu_speed, targets)
   end
 
-  def aggregate_logical_cpus(targets = nil)
-    aggregate_hardware(:hosts, :logical_cpus, targets)
+  def aggregate_cpu_total_cores(targets = nil)
+    aggregate_hardware(:hosts, :cpu_total_cores, targets)
   end
+  alias_method :aggregate_logical_cpus, :aggregate_cpu_total_cores
+  Vmdb::Deprecation.deprecate_methods(self, :aggregate_logical_cpus => :aggregate_cpu_total_cores)
```

Comment:
It is a virtual column so it is automatically exposed through automate.  Not sure about rest.